crates/ruff_linter/src/rules/flake8_commas/rules/trailing_commas.rs

```diff
@@ -319,13 +336,15 @@ fn check_token(
     context: Context,
     locator: &Locator,
     lint_context: &LintContext,
+    settings: &LinterSettings,
 ) {
     // Is it allowed to have a trailing comma before this token?
     let comma_allowed = token.ty == TokenType::ClosingBracket
         && match context.ty {
             ContextType::No => false,
             ContextType::FunctionParameters => true,
             ContextType::CallArguments => true,
+            ContextType::TypeParameters => is_trailing_comma_type_params_enabled(settings),
```

Comment:
I don't think we need the preview check here. The only way we can get a `ContextType::TypeParameters` is if the other preview check was true.


```suggestion
            ContextType::TypeParameters => true,
```